packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -0,0 +1,432 @@
+
+##### Example
+
+```json
+  "type": {
+    "id": "otyfnjfba4ye7pgjB0g4"
+  }
+```
+
+## User Schema Operation Extensions to Non-Default Types
+
+All operations documented under [User Schema Operations](/docs/reference/api/schemas/#user-schema-operations) will continue to operate as before. All these APIs now have an alternate form - replace the trailing `/default` with a schema ID - which will perform the relevant operation on the schema associated with any type.
+
+Just as a schema is currently associated with a User Type, the schema ID is related to the type ID: replace the leading `oty` with `osc`. For example, a User Type with ID `oty1234567890abcdefg` would have a Schema ID of `osc1234567890abcdefg`.
+
+In the future the linkage between Schema and User Type may be extended (for example, to allow multiple Types to share a Schema) but for now this is a 1:1 relationship.
+
+### Example
+
+With the existing Schemas API without User Types, if you want to add a new property to the schema you would make a POST to `/api/v1/meta/schemas/user/default` (updating the default because that is the only type).
+
+With the User Types API, if you created a new type `oty1234567890abcdefg` and later wish to add a property to its schema, send a POST to the same endpoint, replacing `default` with the appropriate Schema ID: `/api/v1/meta/schemas/user/osc1234567890abcdefg`.
+
+Everything else - body of the payload, return values, etc. - is the same as the [Schemas API](/docs/reference/api/schemas/).
+
+## Linked Object endpoint updates
+
+Linked Objects now apply to every User Type. If an org has the default type and 2 custom types, a Linked Object will be capable of linking a User of any of those 3 types to any other User. Each type sees exactly the same Linked Object definitions.
+
+As such, the reference to `/default` in the Linked Objects URL is now deprecated. All old URLs from the [Linked Objects API](/docs/reference/api/linked-objects/) will continue to work for backwards compatibility purposes. But the operations now reside at the new URLs (formed by removing `/default`).
+
+Deprecated URL: `/api/v1/meta/schemas/user/default/linkedObjects`
+New URL: `/api/v1/meta/schemas/user/linkedObjects`
+
+### Example
+
+To create a Linked Object Definition, you can continue to POST to `/api/v1/meta/schemas/user/default/linkedObjects`.
+
+You should POST to the new URL `/api/v1/meta/schemas/user/linkedObjects` instead.
+
+Everything else - body of the payload, return values, etc. - is unchanged. This is simply a change of the URL endpoint.
+
+## User object updates
```

Comment:
Yeah, I wasn't sure how much if any of this should stay on this page once I had written the section in the Users API.

I'll add the following under the Request Parameters for the Update User section of the Users API, and then will remove the section here.  (I'll also remove the link to this section from the "Specify the User Type of a New User" section, and just let it link to the User Model on the Users API page.)

>Note: The User Type of a user cannot be changed. If the Request Parameters include the `type` element from the [User Model](#user-model), the value must match the existing type of the user. To change a User's type, the User object must be deleted and recreated with the desired Type.

Question: I'm considering adding the word "Currently" as in "Currently, the User Type of a user cannot be changed."  That's because several people have observed that we will be asked/expected to provide that functionality eventually, so maybe adding that one word will reduce the number of people asking/griping about it.  Thoughts?

Reply:
Especially considering we already have plans to work on this, I would be in favor of adding the word currently. My concern here would normally be suggesting something we might not have plans on working on, but in this case it's definitely something that needs to be coming down the pipeline.

Reply:
Yeah, it seems to be a requirement for GA, so I'll add "Currently," for now.